packages/ra-core/src/controller/details/useCreateController.ts

```diff
@@ -213,6 +215,10 @@ export const useCreateController = <
     });
 
     return {
+        hasCreate,
```

Comment:
so I don't think the controller should return that data. Same for other controllers